### PR TITLE
Add scripts to check unmaintained repos

### PR DESCRIPTION
**Pull request by zhihao11ui**

### CODE REVIEW

File: auto_check_unmaintained/example/py-data_with_last_commit.csv
Comment (darko-marinov) [inline auto_check_unmaintained/example/py-data_with_last_commit.csv:1]: We don't need automatically generated files in the repo.

File: auto_check_unmaintained/example/unmaintained-repos.csv
Comment (darko-marinov) [inline auto_check_unmaintained/example/unmaintained-repos.csv:1]: We don't need automatically generated files in the repo.

**Review by darko-marinov — CHANGES_REQUESTED** [review]
Please remove the (relatively big) automatically generated files from the PRs. You can `scp` them to your VM and let Ajitesh know where to inspect.

**Comment by Ajitesh27** [comment]
LGTM.
You can raise another PR updating unmaintained tests with their status.

**Pull request merged**